### PR TITLE
Metrics IDB and Ignition Archive, fixes and improvements

### PR DESCRIPTION
**Pull request by Joshlha**
- Metrics IDB charts now have a right click option to pop-out into a new window
- Project exports from the archive tool now export to a single .zip file which can be easily imported into Ignition
- "Save As" feature now works as intended
- Removed "Save As" ability when right clicking a directory.
- Fixed crash when metrics idb contained duplicate datapoints (same value & timestamp)

### CODE REVIEW

File: src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt [hunk 1]
```diff
@@ -130,10 +131,11 @@ class ZipView(path: Path) : ToolPanel("ins 6, flowy") {
                     }
                     add(openIndividually)
 
-                    if (selectedPaths.size == 1) {
+                    val selectedNode = selectedPaths.first().lastPathComponent as PathNode
+
+                    if (selectedPaths.size == 1 && selectedNode.isLeaf) {
```
Comment (paul-griffith) [inline src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt:136]: You might be able to express this slightly cleaner with `selectedPaths.singleOrNull()`, but this is totally fine too

File: src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt [hunk 2]
```diff
@@ -148,8 +150,18 @@ class ZipView(path: Path) : ToolPanel("ins 6, flowy") {
...
         add(bundleInfo, "split 2, height 32!")
-        add(FlatScrollPane(fileTree), "pushy, growy, width 250!")
-        add(tabStrip, "newline, push, grow, spany")
+        add(
+            JSplitPane(
+                JSplitPane.HORIZONTAL_SPLIT,
+                FlatScrollPane((fileTree)),
+                tabStrip,
+            ).apply {
+                dividerSize += 2
```
Comment (paul-griffith) [inline src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt:159]: Just padding to make it easier to grab?

Comment (Joshlha) [reply src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt:159]: Yeah lol. OneClickExpand arrows don't have any space around them and it looks tacky. This just puts more space there.

File: src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt (hunk 1 above)
Comment (Joshlha) [reply src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt:136]: Hm, definitely not following on this logic. The isLeaf check removes the option from folders.

Comment (paul-griffith) [reply src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt:136]: As in, something like this:
```suggestion
                    val selectedNode = selectedPaths.singleOrNull()?.lastPathComponent as? PathNode

                    if (selectedNode?.isLeaf == true) {
```

Not a big deal at all, though.

**Review by paul-griffith — APPROVED** [review]

**Pull request merged**